web/package-lock.json

```diff
@@ -23,11 +23,11 @@
                 "chai-html": "^3.0.0",
                 "copy-webpack-plugin": "^12.0.2",
                 "cross-env": "^7.0.3",
-                "eslint": "^8.57.0",
+                "eslint": "^9.6.0",
                 "eslint-config-prettier": "^9.1.0",
                 "eslint-plugin-jsdoc": "^48.5.2",
                 "eslint-plugin-prettier": "^5.1.3",
-                "eslint-plugin-react-hooks": "^4.6.2",
+                "eslint-plugin-react-hooks": "^5.1.0-rc",
```

Comment:
Do we really want this? Is this mayhaps why Dependabot wasn't updating yet?

Reply:
Possibly... I didn't know it checked peer dependencies though!

Unfortunately it's the first release with eslint9 support :(